packages/radix-ui-themes/src/components/box.tsx

```diff
@@ -24,7 +25,17 @@ interface BoxProps
 const Box = React.forwardRef<BoxElement, BoxProps>((props, forwardedRef) => {
   const { rest: marginRest, ...marginProps } = extractMarginProps(props);
   const { rest: layoutRest, ...layoutProps } = extractLayoutProps(marginRest);
-  const { className, asChild, display = boxPropDefs.display.default, ...boxProps } = layoutRest;
+
+  const {
+    className,
+    asChild,
+    display = boxPropDefs.display.default,
+    style,
+    ...boxProps
+  } = layoutRest;
+
+  const [layoutClassNames, layoutCustomProperties] = getLayoutStyles(layoutProps);
```

Comment:
Wouldn't all the other layout components like `Flex`, ``Container`, `Section` also need all those changes?

![CleanShot 2023-12-20 at 12 30 51@2x](https://github.com/radix-ui/themes/assets/1539897/b3bf52ae-bfd5-490c-b70f-aa4b90533fd6)

Reply:
Yep, adding